### ISSUE TITLE
Alloy Recipe Loading Issue

### ISSUE THREAD

**Issue by charlie-e09ae35d** [MinestrapTeam/Minestrappolation-4#111]
minestrappolation 4.3.0 for minecraft 1.8.9

sorry if i am missing something obvious but is there a reason behind those recipes in alloy smelter?

two apples + two pie crusts = two apple pies, [code link](https://github.com/MinestrapTeam/Minestrappolation-4/blob/1.8.8/src/main/java/minestrapteam/mods/minestrappolation/crafting/recipes/AlloyRecipes.java#L54)
one pumpkin + two pie crusts = two pumpkin pies, [code link](https://github.com/MinestrapTeam/Minestrappolation-4/blob/1.8.8/src/main/java/minestrapteam/mods/minestrappolation/crafting/recipes/AlloyRecipes.java#L53)

**[SEE BELOW DESCRIPTION FOR OVERARCHING ISSUE]**

**Comment by Delocuro** [comment]
Yeah, that's a known issue. It's actually due to some code limitations in the Alloy Furnace's recipe loader; when it's loading recipes, it only takes into account the first instance of any particular item. Thus, if you have two recipes that require the same item, it will just make both of them require the quantity of items specified in whichever one was loaded first -- ie, if you had a "1 apple + 1 pie crust = 1 apple pie" recipe, and a "1 Pumpkin + 2 Pie Crust = 2 Pumpkin Pie" recipe, the pumpkin one would only require 1 pie crust since it was specified in the apple recipe, despite the pumpkin one calling for 2.

Because we hadn't gotten around to fixing that when we added the pie recipes, and I wanted the pumpkin one to give two pies per pumpkin to compensate for the added expensiveness of the pie crust, I had to find some workaround -- hence that weirdness. We really should get that alloy loading issue fixed though, so I'll change the title of this issue to that.
